### ISSUE TITLE
Fagbevegelsen! Fagbevegelsen! Fagbevegelsen!

### ISSUE THREAD

**Issue by danmichaelo** [realfagstermer/realfagstermer#396]
Kommentar fra nynorsk-oversetterne på [Fagbevegelsen](https://trans.biblionaut.net/concepts/REAL006978?view=nn&q=has:editorialNote,graph:local):

> Til bibsysfolket: Skal verkeleg Martin Kolbergs landsmøtepunchline frå 2005 (Fagbevegelsen! x 3) stå som oppslagsord i systemet? Altså, det er heilt greitt for oss (me tykkjer det er sjukt lol :)), men me tenkte det kunne vera greitt å seia frå uansett …

Bibsysfolket, det tror jeg er oss :D

**Comment by Hypsibius** [comment]
Vel, Fagrørsla! Fagrørsla! Fagrørsla! kling vel greitt? Eller så er det noko vi kan fjerne med ein fuktig klut ;-)

**Comment by bibliomari** [comment]
Fnis. Sykt lol.

**Comment by bibliomari** [comment]
Det ER jo gøy, da.... Hvem av oss er det som er så morsom? @BioHeidi @Violabibaluba @knuthe @danmichaelo

**Comment by BioHeidi** [comment]
*Peker på sidemannen*

**Comment by knuthe** [comment]
Vi må nok legge inn samme funksjonalitet som i Alma, så vi kan se hvem som har gjort hva.

**Comment by bibliomari** [comment]
Ja, vi må kunne spore alt tilbake til et ferdig kartlagt gensett!!

**Comment by Hypsibius** [comment]
Genetic barcoding?

**Comment by BioHeidi** [comment]
Barcoding? Det skal jeg, @bibliomari og @danmichaelo drive med i RF-kjelleren etter at Mari og jeg har tatt software carpentry-kurset. Alle kan bli med.

**Comment by Hypsibius** [comment]
RII?

**Comment by bibliomari** [comment]
RII = Roald 2

**Comment by Hypsibius** [comment]
Såklart!
